copy-file-on-save.el

```diff
@@ -125,9 +125,13 @@
 
 (defun copy-file-on-save--copy-file ()
   "Copy a file using `copy-file'."
-  (let ((from-path buffer-file-name)
-        (to-path   (copy-file-on-save--replace-path buffer-file-name)))
-    (copy-file from-path to-path t)))
+  (let* ((from-path buffer-file-name)
+	 (to-path   (copy-file-on-save--replace-path buffer-file-name))
+	 (to-dir (file-name-directory to-path)))
+    (progn
+      (if (not (file-exists-p to-dir))
+	  (make-directory to-dir t) nil)
```

Comment:
You can use `(unless 〜)` instead `(if (not …) 〜 nil)`.

Reply:
Thanks for your idea.